src/device_trezor/trezor/protocol.cpp

```diff
@@ -1017,14 +1012,23 @@ namespace tx {
       }
     }
 
-    // CLSAG support comes here once it is merged to the Monero
-    m_ct.rv->p.MGs.reserve(m_ct.signatures.size());
-    for(size_t i = 0; i < m_ct.signatures.size(); ++i) {
-      rct::mgSig mg;
-      if (!cn_deserialize(m_ct.signatures[i], mg)) {
-        throw exc::ProtocolException("Cannot deserialize mg[i]");
+    m_ct.rv->p.CLSAGs.reserve(m_ct.signatures.size());
```

Comment:
This is technically correct, though it looks like it should be inside the if below, and the reserve for MGs restored in the else.

Reply:
Aah correct. I've fixed it. Thanks!